$OpenBSD: patch-libproxy_CMakeLists_txt,v 1.5 2017/04/17 20:45:38 sthen Exp $

Adding -pthread:
https://code.google.com/p/libproxy/issues/detail?id=196&thanks=196&ts=1370457244

Changing _POSIX_C_SOURCE, adding _BSD_SOURCE: fix with clang/libc++
/usr/include/c++/v1/__bsd_locale_fallbacks.h:51:12: error: use of undeclared identifier 'wcsnrtombs'
etc.

--- libproxy/CMakeLists.txt.orig	Mon Jan  9 14:22:22 2017
+++ libproxy/CMakeLists.txt	Sun Apr 16 22:16:52 2017
@@ -6,8 +6,9 @@ set_project_version(0 4 14)
 if (WIN32)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS=1)
 else(WIN32)
-  add_definitions(-D_POSIX_C_SOURCE=200112L)
-  set(CMAKE_CXX_FLAGS "-fvisibility=hidden ${CMAKE_CXX_FLAGS}")
+  add_definitions(-D_POSIX_C_SOURCE=200809L)
+  add_definitions(-D_BSD_SOURCE)
+  set(CMAKE_CXX_FLAGS "-fvisibility=hidden -pthread ${CMAKE_CXX_FLAGS}")
 endif(WIN32)
 
 ### Modules
